src/benchmarklib/abstract_benchmark_item_runner.cpp

```diff
@@ -38,6 +38,18 @@ void AbstractBenchmarkItemRunner::load_dedicated_expected_results(
   }
 }
 
+bool AbstractBenchmarkItemRunner::has_item_without_dedicated_result() {
+  if (items().size() > _dedicated_expected_results.size()) {
```

Comment:
Regarding what we discussed in person yesterday: does that mean that there is not an element in `_dedicated_expected_results` per item?

Reply:
`_dedicated_expected_results` is only a sparse vector if `load_dedicated_expected_results` was called. If it's not called, `load_dedicated_expected_results` is empty. The load-function is only called during the TPC-DS execution.

Reply:
I understand. Does that mean, we could also check for `_dedicated_expected_results.empty()`?

Reply:
The function name could be misleading then: if no items exists and thus no dedicated result, `has_item_without_dedicated_result` would return true.

Reply:
Right. We could add a comment to the current version of the method or to the header file where `_dedicated_expected_results` is declared, saying that we either have an empty or a sparse vector with the same size as `items()` to make clear that `_dedicated_expected_results.size()` is either `0` or `items().size()`.